### ISSUE TITLE
Client-Server Version Mismatch

### ISSUE THREAD

**Issue by Shadowen** [vjurenka/BWMirror#22]
I can't seem to get the latest versions of BWAPI and BWMirror to work with each other.

I keep getting the following error

```
Error: Client and Server are not compatible!
Client Revision: XXXX
Server Revision: YYYY
```

The advice given (previous issues and readme, etc.) is to update both BWMirror and BWAPI. I have tried this and after a bit of experimentation, I've come to the following conclusions:

**Client - BWMirror**
2.5 = 4709
2.4 = 4615

**Server - BWAPI**
4.2 = 4708
4.1.1B = 4676
4.1.0B_2 = 4615

As you can see, there are no client-server version matches since BWAPI4.1.0B_2 and BWMirror 2.4. Am I missing a link to a release, or have the version numbers simply been mismatched? Thanks.

**Comment by p996** [comment]
They seem to be mismatched.  I'm experiencing the same issue.

**Comment by vjurenka** [comment]
Hi,
thanks for the report, I have fixed the 2.5,

If you are upgrading the jar file from the previous v2.5, delete the old bwapi_bridge2_5.dll so the fixed one will be auto-extracted from the new jar.

Let me know if that works for you

**Comment by p996** [comment]
The new version is working well.  Thanks for the quick fix.
